### ISSUE TITLE
Add forward chain rules to support Diffractor

### ISSUE THREAD

**Issue by gdalle** [gdalle/ImplicitDifferentiation.jl#74]
Not yet crucial because Diffractor is only compatible with Julia 1.10

**Comment by gdalle** [comment]
This is blocked by https://github.com/JuliaDiff/AbstractDifferentiation.jl/issues/80 at the moment

**Comment by gdalle** [comment]
Not a priority, Diffractor is too unstable atm
